paltas/Sampling/sampler.py

```diff
@@ -114,6 +117,101 @@ def sample(self):
 					CROSSOBJECTWARNING = False
 				full_param_dict[component+'_parameters'][param] = (
 					cross_dict[cross_param])
-
+				
 		# Populate the cross objects
 		return full_param_dict
+	
+	def get_sample_dist(self):
```

Comment:
Every function needs a docstring, please! Same format as the rest of the package.